### DISCUSSION TITLE
建议添加右下角图标自动放大

### DISCUSSION THREAD

**Discussion by Linux-qitong** [linuxdeepin/developer-center#4496] [Features Request & Ideas | 特性请求 & 头脑风暴]
https://bbs.deepin.org/phone/zh/post/257241
![20230526100438772_image](https://github.com/linuxdeepin/developer-center/assets/130018882/eb5053b8-046d-4dba-b845-fcfd2b4f147c)
坛友反应无法放大。

**Comment by Decodetalkers** [comment]
什么无法放大？

**Reply by Linux-qitong** [reply]
右下角的系统图标托盘

**Reply by BLumia** [reply]
托盘图标放大的话，如果提供托盘的第三方程序给了很小的图标（比如 QQ 的托盘），直接放大不是会非常糊吗？这种是预期的显示方式吧？还是说这个建议里面预期的行为是别的形式，比如够显示双行就双行之类？

**Reply by Linux-qitong** [reply]
坛友预期托盘图标放大，任务栏放大图标托盘比较小。

**Reply by BLumia** [reply]
还是上面我提到的这个问题：

> 托盘图标放大的话，如果提供托盘的第三方程序给了很小的图标（比如 QQ 的托盘），直接放大不是会非常糊吗？
